src/main/java/org/jabref/logic/openoffice/oocsltext/CSLCitationOOAdapter.java

```diff
@@ -59,11 +58,11 @@ public static void writeCitation(XTextDocument doc, XTextCursor cursor, String c
      * The transformed HTML can be used for inserting into a LibreOffice document
      * Context: The HTML produced by CitationStyleGenerator.generateCitation(...) is not directly (completely) parsable by OOTextIntoOO.write(...)
      * For more details, read the documentation of the write(...) method in the {@link OOTextIntoOO} class.
-     * Additional information: https://devdocs.jabref.org/code-howtos/openoffice/code-reorganization.html.
+     * Additional information: <a href="https://devdocs.jabref.org/code-howtos/openoffice/code-reorganization.html">...</a>.
```

Comment:
This is wrong. Either repeat the link inside the a tag or use "additional information" instead of ...

Reply:
Should I make a separate PR or club it with PR-C of my project?
I accepted IntelliJ's suggestion blindly, forgetting how html hrefs work🤦🏻